client/layout/sidebar/skip-navigation.jsx

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import PropTypes from 'prop-types';
+import { localize } from 'i18n-calypso';
+import React from 'react';
+
+class SkipNavigation extends React.Component {
+	static propTypes = {
+		skipToElementId: PropTypes.string,
+	};
+
+	render() {
+		return (
+			<a
+				href={ '#' + this.props.skipToElementId }
+				className="sidebar__skip-navigation"
+				data-tip-target={ this.props.tipTarget }
+			>
+				{ this.props.translate( 'Skip Navigation' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **46** times:
`translate( 'Skip navigation' )` *ES Score: 16*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/pull/22791)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).